content/docs/command-reference/plots/index.md

```diff
@@ -260,3 +277,48 @@ file:///Users/usr/src/plots/classes.csv.html
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
 > predefined in DVC (found in `.dvc/plots/confusion.json`).
+
+### Example for the HTML template
+
+Download the Vega libraries into the directory where you'll produce the
+`dvc plots`.
+
+```dvc
+$ wget https://cdn.jsdelivr.net/npm/vega@5.10.0 -O my_vega.js
+$ wget https://cdn.jsdelivr.net/npm/vega-lite@4.8.1 -O my_vega_lite.js
+$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.5.1 -O my_vega_embed.js
+```
+
+Create the following HTML file with your text editor:
+
+```html
+<html>
+  <head>
+    <script src="my_vega.js" type="text/javascript"></script>
+    <script src="my_vega_lite.js" type="text/javascript"></script>
+    <script src="my_vega_embed.js" type="text/javascript"></script>
+  </head>
+  <body>
+    {plot_divs}
+  </body>
+</html>
+```
+
+Note that this is a standard HTML file with only `{plot_divs}` as a placeholder
+for the generated plots. `<script>` tags in this file point to the local
+JavaScript libraries we downloaded above. Supposing this template was named
+`.dvc/page_template.html`, we can use it like this:
```

Comment:
> .dvc/page_template.html

Is that a recommended place to put it? What about just the repo root, or .dvc/tmp/plot/templates/ cc @pared

Reply:
The earlier example by @pared used this path but it's not mandatory.

Reply:
I mention the `<script>` tags a few times because they are the really important part here. They should point to the files we downloaded and that may be a bit tricky.

Reply:
I would use path `.dvc/plots/` e.g.:

```suggestion
`.dvc/plots/myreport.html`, we can use it like this:
```

Since that's what's used for https://dvc.org/doc/command-reference/plots#custom-templates too.

Reply:
I'll move this to a specific suggestion too. Resolving